wandb/util.py

```diff
@@ -845,9 +856,12 @@ def prompt_api_key(api, input_callback=None, browser_callback=None, no_offline=F
     # If we're not in an interactive environment, default to dry-run.
     elif not isatty(sys.stdout) or not isatty(sys.stdin):
         result = LOGIN_CHOICE_DRYRUN
+    elif local:
+        result = LOGIN_CHOICE_EXISTS
     else:
```

Comment:
This is the only actual change in this file.  We add a local option which gets rid of signup and dryrun.